NitroxServer/GameLogic/Entities/NitroxEntitySlot.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NitroxServer.GameLogic.Entities
+{
+    public class NitroxEntitySlot
+    {
+        public string[] AllowedTypes;
+
+        public string BiomeType;
+
```

Comment:
It doesn't look like these are used.  Should they be?  Or are we just doing this to ensure the parsing cursor moves?

Reply:
this is in place for something im finishing in the near future same for the dead code above to allow EntitySlot spawning under prefab placeholders